WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/OrderPaymentSection.swift

```diff
@@ -246,7 +246,7 @@ private extension OrderPaymentSection {
     }
 
     var rowsEditImage: Image? {
```

Comment:
Since we've added the lock now, `rowsEditImage` cannot be longer `nil`, right? 

```suggestion
    var rowsEditImage: Image {
```